docker-compose.yml

```diff
@@ -6,11 +6,11 @@ services:
     ports:
       - "5000:5000"
     environment:
-      # Both the api service and the database
+      # Option 1. Full docker local environment (application + postgres)
       - "DB_URI=postgresql+psycopg2://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST_LOCAL_DOCKER}:${DATABASE_PORT_LOCAL_DOCKER}/${DATABASE_LBS_NAME}"
       - "LOGS_DB=postgresql+psycopg2://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST_LOCAL_DOCKER}:${DATABASE_PORT_LOCAL_DOCKER}/${DATABASE_LOGGER_NAME}"
 
-      #or for local docker (API locally, the db remotely)
+      # Option 2. Only application container, the database is hosted somewhere else
```

Comment:
Asking user to uncomment the correct part of docker-compose.yml file depending on how he wants to use database server looks weird. Are you sure, it is a good decision?